src/Pact/Repl/Lib.hs

```diff
@@ -424,5 +430,15 @@ setGasRate :: RNativeFun LibState
 setGasRate _ [TLitInteger r] = do
     setenv (eeGasEnv . geGasModel) (constGasModel $ fromIntegral r)
     return $ tStr $ "Set gas rate to " <> tShow r
-
 setGasRate i as = argsError i as
+
+-- | This is the only place we can do an external call to a capability,
+-- using 'evalCap False'. This allows us to call magical capabilities
```

Comment:
Let's ditch the "This allows us to ... within the coin contract code"

Reply:
👍